README.md

```diff
@@ -77,6 +74,11 @@ npm start
 ```
 
 ### Running the tests
+
 ```sh
 npm test
 ```
+
+## API components
+
+This component is a part of [API components ecosystem](https://elements.advancedrestclient.com/)
```

Comment:
This link seems to be dead btw. Maybe it should be changed or removed?

Reply:
I am removing this line whenever I update a component.